mrt/meetings/participant.py

```diff
@@ -615,6 +618,22 @@ def get(self):
             data['verified'] = 'Yes' if p.verified else None
             data['credentials'] = 'Yes' if p.credentials else None
 
+            for custom_field in added_custom_fields:
+                dat = _CUSTOM_FIELDS_MAP[custom_field.field_type.code]
```

Comment:
Using form fields outside a form is not recommended. Why not take the values from the database?